src/cls/rbd/cls_rbd.cc

```diff
@@ -7075,6 +7075,48 @@ int group_dir_add(cls_method_context_t hctx, bufferlist *in, bufferlist *out)
   return group::dir_add(hctx, name, id, true);
 }
 
+/**
+ * Get a group info.
+ *
+ * Input:
+ * @param name (std::string)
+ *
+ * Output:
+ * @param string of group id
+ * @return 0 on success, negative error code on failure
+ */
+int group_dir_info(cls_method_context_t hctx, bufferlist *in, bufferlist *out)
```

Comment:
I don't think a new class method is needed -- the group ID can be obtained by calling existing `dir_get_id` method on `rbd_group_directory` object in the group's pool.